sdf_timing/utils.py

```diff
@@ -1,3 +1,42 @@
+import re
+
+
+def get_scale(timescale):
+    """Convert sdf timescale to scale factor
+
+    >>> get_scale('1.0 fs')
+    1e-15
+
+    >>> get_scale('1ps')
+    1e-12
+
+    >>> get_scale('10 ns')
+    1e-08
+
+    >>> get_scale('10.0 us')
+    9.999999999999999e-06
+
+    >>> get_scale('100.0ms')
+    0.1
+
+    >>> get_scale('100 s')
+    100.0
+
+    """
+    mm = re.match(r'(10{0,2})(\.0)? *([munpf]?s)', timescale)
+    sc_lut = {
+        's': 1.0,
+        'ms': 1e-3,
+        'us': 1e-6,
+        'ns': 1e-9,
+        'ps': 1e-12,
+        'fs': 1e-15,
+    }
+    ret = None
```

Comment:
If there is no match, you get none back? Is that the right thing to do? Or should it raise an error?

Reply:
Probably a good idea. I'll change to follow errors in the rest of the package.

Reply:
Done